chocolatey/choco
# (maint) Replace .editorconfig with nuget package

## Description Of Changes

- Remove editorconfig file
- Add gitignore entry
- Install nuget package for the editorconfig file/best practices

## Motivation and Context

In order for us to have a consistent baseline for code style, we've pulled our standards out into a Chocolatey.BestPractice nuget package, which is being brought in to here.

## Testing

1. Build the project in VS to populate the nuget package and files
2. Verify the editorconfig rules are still being applied by adding or modifying code to cause a violation. (for example, I tested these rules are still in effect by adding a single-line if statement without braces and checking VS's error list to ensure the IDE0011 style rule is flagged as it should be with our editorconfig settings.

### Operating Systems Testing

N/A

## Change Types Made



## Change Checklist


## Related Issue
N/A
